src/pages/shipping.js

```diff
@@ -175,13 +175,16 @@ export const startShipping = async (userName) => {
         </p>
     </div>
     <div id="edit">
-        <div class="row">
-          <div class="col-5 no-gutters">
-            <h4 style="text-align:center;">Boxes in Process</h4>
+        <div class="row" style="margin-bottom:.5rem">
+          <div class="col-9 no-gutters">
+            <h4 style="text-align:start;">Select one or more boxes to ship</h4>
+          </div>
+          <div class="col-3 no-gutters">
+          <button type="button" class="btn btn-primary" data-dismiss="modal" id="completePackaging" style="margin:auto;display:block;">Continue to Shipping Manifest</button>
           </div>
         </div>
         
-            <table  class="table" style="width:100%;border:1px solid;" id = "saveTable">
+            <table  class="table table-bordered" style="width:100%;border:1px solid;" id = "saveTable">
```

Comment:
I think we need to nest this table element inside a div element to apply a solid border on this table. 

<div style="border: 1px solid black; overflow: auto; margin-bottom: 0.5rem; height: 400px;">
</div>